scripts/prepare-tree-import.mjs

```
@@ -0,0 +1,174 @@
+      }
+
+      const recordTypeId = recordTypeMap
+        .get(record.RecordType.SobjectType)
+        .get(record.RecordType.DeveloperName);
+      if (recordTypeId == null) {
+        throw new Error(
+          `Could not find RecordType with SobjectType ${record.RecordType.SobjectType} and DeveloperName ${record.RecordType.DeveloperName}. Processing ${JSON.stringify(record)}`
+        );
+      }
+
+      delete record.RecordType;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)